client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx

```diff
@@ -174,42 +173,31 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
                 </Grid>
                 <Grid item>
                     <Grid container justify='space-between'>
-                        <FieldName xs={7} fieldName={RELEVANT_OCCUPATION_LABEL}/>
-                        <Grid item xs={5}>
+                        <FieldName xs={5} fieldName={OCCUPATION_LABEL}/>
```

Comment:
the sizes switched in purpose?

Reply:
yes, the drop down is different size from radio button, and also the label has become shorter